### DISCUSSION TITLE
[SANDBOX] Impossible créer une configuration de paiement

### DISCUSSION THREAD

**Discussion by LuisCruzRamon** [PayGreen/documentation#6] [Q&A] [answered]
Bonjour,

Il m'est impossible de créer une configuration de paiement via [https://developers.paygreen.fr/reference/post_create_payment_config](https://developers.paygreen.fr/reference/post_create_payment_config) avec le but de créer des paiements pour carte bancaire.

J'ai le message "You must provide a config for the platform bank_card", c'est quoi cette "config" ?  Sur l'API Référence ce n'est pas une information obligatoire.

Merci

**Comment by rapinto** [comment]
Same issue. Can we get some information about this config parameter ?

**Comment by AdrienBr** [comment, marked as answer]
Hello,
Sorry for delay. When creating a `bank_card` Payment Config, you need to set this config : 

```
{
   ...
    "config": {
        "is_vad": true | false
    }
}

```

`true` means that a selling contract exists between the shop and the bank. The funds go straight from the Buyer to the shop bank account.  A Selling Contract object is required in this case, with some more information (https://developers.paygreen.fr/reference/post_create_selling_contract)

`false` means that the shop needs a PayGreen wallet that receives payments. In this case, some compliance documents are required, so the full process must be carried out on the dashboard.

We will clarify our documentation on this subject.
